File: java/src/main/java/java细节大全/泛型/v1/Point.java
package java细节大全.泛型.v1;

/**
 * Created by shejiewei on 2019/3/27.
 */
 class Point<T> {
    private T var;

    public T getVar() {
        return var;
    }

    public void setVar(T var) {
        this.var = var;
    }

     public static void main(String[] args) {

         Point<String> point = new Point<>();

         point.setVar("aa");
         System.out.println(point.getVar());
     }
}

File: spark/src/main/java/类和对象/Point.scala
package 类和对象

/**
  * Created by shejiewei on 2019/5/27.
  */
class Point(xc:Int,yc:Int) {
  var x:Int=xc
  var y:Int=yc

  def move(dx:Int,dy:Int): Unit =
{
   x=x+dx
  y=y+dy
   println("x的坐标点:"+x)
   println("y的坐标点:"+y)

}
}
